docs/link.md

```diff
@@ -0,0 +1,39 @@
+# link
+
+#### value:
+
+Value of url field.
+
+- Type: `String`
+- Required: No
+
+#### onChange:
+
+A function that receives the value of the new option that is being selected as input. It passes the new `value` as first argument and `props` as second argument.
+
+- Type: `function`
+- Required: No
+
+#### label:
+
+Label for input but applicable only when using it in inspector controls.
+
+- Type: `String`
+- Required: No
+
+For more read gutenberg [readme](https://github.com/WordPress/gutenberg/tree/master/blocks/url-input).
+
+**Example:**
+
+```js
+url: {
+	type: 'string',
+	field: {
+		type: 'link',
+	},
+}
+```
+
+### Return
+
+- Type: `string` - Print link
```

Comment:
I think "Print" doesn't look like an appropriate word here. Because the value isn't printed, its returned.

Reply:
Updated.